packages/fcl/src/authentication/README.md

```diff
@@ -26,10 +26,10 @@ console.log("The Current User", currentUser)
 ### Subscribe to Current User
 
 ```javascript
-import * as fcl from "@onflow/fcl"
+import fcl from '@onflow/fcl';
 
-// Returns an unsubscribe function
-const unsubscribe = fcl.currentUser()
+// Returns a subscribe function
```

Comment:
The previous code is corrent. Calling the function returned from `fcl.currentUser()` will remove the event listener. It will 'unsubscribe' from the ... subscription.

eg
```js
// function passed as callback to subscribe will no longer get called when current user updates.
unsubscribe()
```

Reply:
🤦‍♀️ duly noted. I was confused by the wording of things. Thought it was a typo!